source/robot_model/tests/testRobotModel.cpp

```diff
@@ -91,8 +91,22 @@ TEST_F(RobotModelTest, TestJacobianNbCols) {
   EXPECT_EQ(jac.get_nb_cols(), joint_state.get_size());
 }
 
-TEST_F(RobotModelTest, TestType) {
+TEST_F(RobotModelTest, TestComputeCoriolisMatrix) {
+  StateRepresentation::JointState js = StateRepresentation::JointState::Random("robot", 7);
+  Eigen::MatrixXd coriolis = franka.compute_coriolis_matrix(js);
+  EXPECT_TRUE(coriolis.rows() == js.get_size() && coriolis.cols() == js.get_size());
+}
+
+TEST_F(RobotModelTest, TestComputeCoriolisForces) {
+  StateRepresentation::JointState js = StateRepresentation::JointState::Random("robot", 7);
+  StateRepresentation::JointTorques coriolis_forces = franka.compute_coriolis_forces(js);
+  EXPECT_TRUE(coriolis_forces.data().norm() > 0);
```

Comment:
For a known robot model in the test fixture, these tests should include specific numerical cases. Some known q, v configuration and check that the matrix / joint torques are correct for both gravity and coriolis. We can use matlab toolbox or other known-good solvers to verify. Though I trust the pinocchio library and that it's being used correctly here, I also want to ensure that we don't make any dangerous assumptions. Sanity checking that the robot model, input and output is all parsed correctly, reference frame for this torque (if this torque is positive / negative), if the coriolis matrix is pre or post-multiplied correctly, and so on.

Reply:
Yes I agree we should find some good test fixtures.